api/cwatch_response.php

```diff
@@ -1,17 +1,17 @@
 <?php
 
-class ApiResponse 
+class CwatchResponse
```

Comment:
This class would be much better if it did not use public properties and used public methods instead.

```
public function status(); // whether the response was successful or not. might be http code, boolean, or something
public function raw(); // return the raw API response
public function response(); // converts the API response to object (json_decode, and on failure return a blank object)
public function errors(); // retrieves errors, if any
```